src/legacy/core_plugins/interpreter/public/functions/esaggs.ts

```diff
@@ -99,7 +106,7 @@ export const esaggs = (): ExpressionFunction<typeof name, Context, Arguments, Re
     searchSource.setField('index', indexPattern);
     searchSource.setField('size', 0);
 
-    const response: Pick<KibanaDatatable, 'columns' | 'rows'> = await courierRequestHandler({
```

Comment:
FWIW, I don't remember exactly, but I *think* this was here originally to avoid having to use `column: any` on line 125.

Reply:
Yes, but the response of `courierRequestHandler` is more than a regular datatable (e.g. there are `aggConfig` objects on the columns), that's why switching from untyped to typed makes problems in the new column map function below. I typed the courier request handler a bit, makes this file cleaner.